ui/src/polyfills.ts

```diff
@@ -18,17 +18,18 @@
  * BROWSER POLYFILLS
  */
 
+//see https://github.com/angular/angular-cli/issues/4654
 /** IE9, IE10 and IE11 requires all of the following polyfills. **/
-// import 'core-js/es6/symbol';
```

Comment:
Based on a verbal conversation.  We needed to un-cmment some of the imports for phantom.js headles browser for testing.  The remaining commented out code is for retrospective requirements.